source/intro.Rmd

```diff
@@ -297,32 +346,31 @@ tonight at the local cinema; you can look it up on the web or call the cinema
 and ask. But consider by contrast how difficult it is to determine with
 accuracy:
 
-a. what will be the result of more than a hundred million Americans voting for
+1. Whether we will save lives by recommending vitamin D supplements for the
+   whole population as protection against viral infections.  Some evidence
+   suggests that low vitamin D levels predispose to more severe lung
+   infections, and that taking supplements can help [@martineau2017vitamin].
+   But, how certain can we be of the evidence?  How safe are the supplements?
+   Does the benefit, and the risk, differ by ethnicity?
+2. What will be the result of more than a hundred million Americans voting for
    president a month hence; the best attempt usually is a sample of 2000
```

Comment:
hence is weirdly placed in this sentence; correct, but probably not language that younger readers will parse perfectly.

Reply:
Changed to "from now".